src/orb.yml.hbs

```diff
@@ -564,17 +570,20 @@ commands:
                     --cluster << parameters.cluster-name >> \
                     --services ${SERVICE_NAME} \
                     --output text \
-                    --query 'services[0].deployments[].[taskDefinition, status]')
+                    --query 'services[0].deployments[].[taskDefinition, status]' \
+                    --profile << parameters.profile-name >>)
                 NUM_DEPLOYMENTS=$(aws ecs describe-services \
                     --cluster << parameters.cluster-name >> \
                     --services ${SERVICE_NAME} \
                     --output text \
-                    --query 'length(services[0].deployments)')
+                    --query 'length(services[0].deployments)' \
+                    --profile << parameters.profile-name >>)
```

Comment:
Could this be updated such that --profile will not be set if the value of profile-name is ""?